### PR TITLE
{bio,devel,tools}[GCCcore/9.3.0] openCARP v3.2, gengetopt v2.23, texinfo v6.7

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node3123.skitty.os - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/7d0e80da0edd2861ad76b8368b4f9847 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node3405.kirlia.os - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) Gold 6240 CPU @ 2.60GHz (cascadelake), Python 2.7.5
See https://gist.github.com/c459b8999ff2650646758ccacbd8d126 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node2426.golett.os - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz (haswell), Python 2.7.5
See https://gist.github.com/c33a8aefa2c2811b7750bee7b39e03fc for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
**FAILED**
Build succeeded for 9 out of 10 (3 easyconfigs in this PR)
nuc.localdomain - Linux Fedora 32, x86_64, Intel(R) Core(TM) i7-8650U CPU @ 1.90GHz, Python 3.8.5
See https://gist.github.com/bfed35487fa92dafea38c59b7dd1607b for a full test report.

**Comment by verdurin** [comment]
```
ld.gold: error: cannot find -lLIB_petsc-NOTFOUND
collect2: error: ld returned 1 exit status
make[2]: *** [tools/mesher/CMakeFiles/mesher-bin.dir/build.make:165: bin/mesher] Error 1
```

**Comment by verdurin** [comment]
Test report by @verdurin
**FAILED**
Build succeeded for 15 out of 16 (3 easyconfigs in this PR)
centos8-test0.virt - Linux UNKNOWN UNKNOWN, x86_64, Intel Core Processor (Skylake, IBRS), Python 3.8.0
See https://gist.github.com/87c2b60fc0256918a0afab327462a1fc for a full test report.

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 7 out of 7 (3 easyconfigs in this PR)
node373.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/61b501b720ea5234b18a44aae67fefa2 for a full test report.

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 7 out of 7 (3 easyconfigs in this PR)
node118.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) CPU E5-2680 v2 @ 2.80GHz, Python 2.7.5
See https://gist.github.com/86b664e583db127571b1ab335104c8f2 for a full test report.

File: easybuild/easyconfigs/o/openCARP/openCARP-3.2-foss-2020a-Python-3.8.2.eb [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+patches = ['openCARP-%(version)s_build-info.patch']
+checksums = [
+    '4029f9bb0786636380c25d995ef7be22626daaa2c76bf741dea949df11da7bf7',  # openCARP-v3.2.tar.gz
+    '43004c7dee2981677c4316c32ce20080ad2e70760d53981aaaaba55ebc659e8c',  # openCARP-3.2_build-info.patch
+]
+
+builddependencies = [
+    ('CMake', '3.16.4'),
+    ('gengetopt', '2.23'),
+]
```
Comment (lexming) [inline easybuild/easyconfigs/o/openCARP/openCARP-3.2-foss-2020a-Python-3.8.2.eb:24]:
```suggestion
    ('pkg-config', '0.29.2'),
]
```

**Review by lexming — CHANGES_REQUESTED** [review]
There is a missing dependency on `pkg-config`. Maybe that is what is causing the `not found` errors for PETSc in _generoso_?

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node3411.kirlia.os - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) Gold 6240 CPU @ 2.60GHz (cascadelake), Python 2.7.5
See https://gist.github.com/5018df782ef984e599540e2b9dbfd24a for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node2400.golett.os - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz (haswell), Python 2.7.5
See https://gist.github.com/86cb9dcc657b845e0c14af1202c88967 for a full test report.

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node368.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/e06e99f691cbb4dac3a50365f3b62c6f for a full test report.

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node126.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) CPU E5-2680 v2 @ 2.80GHz, Python 2.7.5
See https://gist.github.com/91b4973fe2301043ec3237e5c274eb31 for a full test report.

**Review by lexming — APPROVED** [review]
LGTM

**Comment by lexming** [comment]
Going in, thanks @boegel !

**Pull request merged**